notebooks/stable-video-diffusion/stable-video-diffusion.ipynb

```diff
@@ -0,0 +1,1300 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #135.    demo.launch()
</pre><p> </p>

<pre class="ql-syntax" spellcheck="false">try:
    demo.queue().launch(debug=True)
except Exception:
    demo.queue().launch(debug=True, share=True)
# if you are launching remotely, specify server_name and server_port
# demo.launch(server_name='your server name', server_port='server port in int')
# Read more in the docs: https://gradio.app/docs/
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
updated